src/tools/rbd/action/Kernel.cc

```diff
@@ -163,6 +163,7 @@ static int parse_unmap_options(char *options)
 
 static int do_kernel_showmapped(Formatter *f)
 {
+#if defined(__linux__)
```

Comment:
@wjwithagen could you add `WITH_KRBD` to `config-h.in.cmake` and conditionalize on it instead?